src/Stratis.Bitcoin.Features.BlockStore/BlockStoreSignaled.cs

```diff
@@ -104,17 +110,56 @@ protected override void OnNextCore(Block block)
             this.logger.LogTrace("(-)");
         }
 
+        /// <summary>Initializes the <see cref="BlockStoreSignaled"/>.</summary>
+        public void Initialize()
+        {
+            this.localBatch = new List<ChainedBlock>();
+
+            // Configure batch timer.
+            this.batchTimer = new Timer(this.batchInterval.TotalMilliseconds);
+            this.batchTimer.AutoReset = false;
+            this.batchTimer.Elapsed += async (sender, args) =>
```

Comment:
can we have a non-anonymous method for these guys?
i'd also create the timer inside constructor and only enabled it here

Reply:
Fixed.